data/data/bootstrap/files/etc/ignition-machine-config-encapsulated.json.template

```diff
@@ -0,0 +1,43 @@
+{
+  "metadata": {
+    "name": "rendered-master-61764a341438250970e46ed30b03faca",
+    "selfLink": "/apis/machineconfiguration.openshift.io/v1/machineconfigs/rendered-master-61764a341438250970e46ed30b03faca",
+    "uid": "e24ef66c-a612-4e6a-93e4-f53386459a76",
+    "resourceVersion": "3754",
+    "generation": 1,
+    "creationTimestamp": "2019-11-01T19:54:59Z",
+    "annotations": {
+      "machineconfiguration.openshift.io/generated-by-controller-version": "093b7ad7090894e844e0a51d8695d94eb24f7719"
+    },
+    "ownerReferences": [
+      {
+        "apiVersion": "machineconfiguration.openshift.io/v1",
+        "kind": "MachineConfigPool",
+        "name": "master",
+        "uid": "5b8a996e-3f43-48f4-aebe-5d10964c1ef7",
+        "controller": true,
+        "blockOwnerDeletion": true
+      }
+    ]
```

Comment:
drop all these, i don't think this is needed.

Reply:
Right, we should indeed be able to drop most of this.  But the MCD will try to deserialize it as a `MachineConfig` object, so it does need the default kube object bits of `kind`, `metadata/name` etc.